provision/examples/azure/main.go

```diff
@@ -20,7 +20,7 @@ func main() {
 	log.SetOutput(ioutil.Discard)
 
 	cluster := &types.Cluster{
-		KubernetesVersion: "1.15",
+		KubernetesVersion: "1.15.7",
```

Comment:
I would like 1.16 here since we kyma supports 1.16 already

Reply:
Done thanks!